source-oracle/replication.go

```diff
@@ -484,6 +534,10 @@ func (s *replicationStream) poll(ctx context.Context) error {
 		// that is small enough to not exceed the limit of files we need
 		var endSCN = startSCN + int64(s.db.config.Advanced.IncrementalSCNRange)
 
+		if maxSCN < endSCN {
+			endSCN = maxSCN + 1
```

Comment:
Is the `+ 1` here because endSCN is exclusive but maxSCN is inclusive, or is there some other reason it's necessary?